doc/man3/SSL_read.pod

```diff
@@ -87,22 +87,15 @@ with the same arguments.
 
 =head1 RETURN VALUES
 
-SSL_read_ex() and SSL_peek_ex() will return 1 for success or 0 for failure.
-Success means that 1 or more application data bytes have been read from the SSL
-connection. Failure means that no bytes could be read from the SSL connection.
-Failures can be retryable (e.g. we are waiting for more bytes to
-be delivered by the network) or non-retryable (e.g. a fatal network error). In
-the event of a failure call L<SSL_get_error(3)> to find out the reason which
-indicates whether the call is retryable or not.
```

Comment:
Why are you deleting this? This is the intended semantics (although actually SSL_read_ex()/SSL_peek_ex() don't currently do this!....but that should be fixed)